src/doppler-types.ts

```diff
@@ -1,4 +1,10 @@
-export type UnexpectedError = { success?: false; message?: string | null; error?: any };
+export type UnexpectedError = {
+  success?: false;
+  message?: string;
+  trace?: any;
+  fullResponse?: any;
+  error?: any;
```

Comment:
Is it not possible adding properties dinamically? Altough maybe it is better having them here, I am not sure.

Reply:
Like having an object of properties for example? 
```
export type UnexpectedError = {
  success?: false;
  properties?: any;
};
```
Then in `properties`:
```
properties.trace= "...";
properties.fullResponse={...};
...
```

Reply:
I thought that it was possible without another object, but I do not know. I had not enough practice with TypeScript

Reply:
I think it's ok to have success explicit the rest of the properties can be dynamic. Like this:
```
export type UnexpectedError = {
  success?: false;
  [key: string]: any;
};
```